sgkit/io/vcf/vcf_reader.py

```diff
@@ -322,12 +324,13 @@ def vcf_to_zarr(
 ) -> None:
     """Convert VCF files to a single Zarr on-disk store.
 
-    For a single input and a single region, the conversion is carried out sequentially.
+    For a single input and a single region, the conversion is carried out sequentially,
```

Comment:
This seems a little confusing since regions and target_part_size can't be specified together.  Perhaps: 

> Conversion is carried out sequentially unless multiple inputs, a target partition size, or multiple regions within at least one input are specified.